tests/phpunit/CRM/Member/Form/MembershipRenewalTest.php

```diff
@@ -379,7 +379,7 @@ public function testSubmitRecurCompleteInstant(): void {
     $log = $this->callAPISuccessGetSingle('MembershipLog', ['membership_id' => $membership['id'], 'options' => ['limit' => 1, 'sort' => 'id DESC']]);
     $this->assertEquals(CRM_Utils_Time::date($nextYear . '-01-01'), $log['start_date']);
     $this->assertEquals(CRM_Utils_Time::date($nextYear . '-01-31'), $log['end_date']);
-    $this->assertEquals(CRM_Utils_Time::date('Y-m-d'), $log['modified_date']);
+    $this->assertApproxEquals(strtotime(CRM_Utils_Time::date('Y-m-d H:i:s')), strtotime($log['modified_date']), 20);
```

Comment:
I think this can be simplified to just `CRM_Utils_Time::time()` but I'll put it up as a followup.